src/libutil/simd_test.cpp

```diff
@@ -1611,8 +1611,8 @@ void test_mathfuncs ()
     VEC expA = mkvec<VEC> (0.367879441171442f, 1.0f, 2.718281828459045f, 90.0171313005218f);
     OIIO_CHECK_SIMD_EQUAL (exp(A), expA);
     OIIO_CHECK_SIMD_EQUAL_THRESH (log(expA), A, 1e-6f);
-    OIIO_CHECK_SIMD_EQUAL (fast_exp(A),
-                mkvec<VEC>(fast_exp(A[0]), fast_exp(A[1]), fast_exp(A[2]), fast_exp(A[3])));
+    OIIO_CHECK_SIMD_EQUAL_THRESH (fast_exp(A),
+                mkvec<VEC>(fast_exp(A[0]), fast_exp(A[1]), fast_exp(A[2]), fast_exp(A[3])), 1e-5f);
```

Comment:
I liked that scalar and simd were required to have identical results, but this would at least solve #4064 without needing to actually make both code paths equal. I'll leave it up to you for which approach you want to take.

Reply:
At some point in my chasing this down, I made this change. Ultimately, disabling the SIMD path for the old MSVS versions in this function took care of the problem, so this change isn't strictly needed to close out this PR. So I'm not totally averse to backing that out, but I'm inclined to keep it since for whatever reason, fast_exp has seemed especially problematic for us historically.

I think my rationale here was that the family of fast_* functions in fmath.h is already advertised as being approximate, deviating slightly from IEEE full precision and not necessarily being bit identical on all platforms (CPU + compiler + build flags). Even on the same platform, you could get ever so slight differences in scalar vs simd results depending on if one used fused mad (fp contraction) but the other didn't, or other minor compiler rearrangements of the underlying instructions.